Src/Sarif.PatternMatcher/Sarif.PatternMatcher.csproj

```diff
@@ -10,18 +10,16 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
```

Comment:
><PackageReference Include="Newtonsoft.Json" Version="12.0.3" /> [](http://example.com/codeflow?start=4&length=63)

This may retrigger the Newtonsoft alert and dependabot.